styles.css

```diff
@@ -145,3 +152,426 @@ body {
   display: flex;
   flex-direction: row;
 }
+
+.Works {
+  display: grid;
+  grid-template-columns: auto;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.About-My {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+}
+
+.Top {
+  display: flex;
+  flex-direction: column;
+}
+
+.screenshot-img {
+  height: 220px;
+  width: 295px;
+  left: 16px;
+  top: 16px;
```

Comment:
- [ ] I suggest that it would be better for sizes not to use fixed units i.e px, but rather responsive ones e.g vh, svh, %. Because using fixed causes your design not to be responsive. in this case, your margins do not match the Figma design. Here is a [video](https://www.youtube.com/watch?v=CyPlIPjoKew) to guide 

### Your design:
![bad margin](https://user-images.githubusercontent.com/102636489/227693783-6f4d881a-a345-4d83-b5fd-6d9d570cbdea.png)

### Figma design
![good margins](https://user-images.githubusercontent.com/102636489/227694015-8f162e6a-1340-4816-ab2e-328d328a784e.png)